server/data/mentions.js

```diff
@@ -3,61 +3,85 @@
     platform: "reddit",
     image: 'https://upload.wikimedia.org/wikipedia/fr/f/fc/Reddit-alien.png',
-    date: Date.now(),
+    date: new Date('October 12, 2020 09:39:09'),
     popularity: 233,
   },
   {
     title: 'PayPal invested $600 in ArcturusCorp and $100 in MercuryCorp',
     content: 'Ut enim ad minim veniam, quis nostrud exercitation ullamco MercuryCorp laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure ArcturusCorp dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.',
     platform: "reddit",
     image: 'https://upload.wikimedia.org/wikipedia/fr/f/fc/Reddit-alien.png',
-    date: Date.now(),
+    date: new Date('October 19, 2020 07:42:09'),
     popularity: 317,
   },
   {
     title: 'PayPal invested $700 in MercuryCorp and $100 in JupiterCorp',
     content: 'Ut enim ad minim veniam, quis nostrud exercitation ullamco JupiterCorp laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure MercuryCorp dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.',
     platform: "reddit",
     image: 'https://upload.wikimedia.org/wikipedia/fr/f/fc/Reddit-alien.png',
-    date: Date.now(),
+    date: new Date('November 12, 2020 02:49:29'),
     popularity: 852,
   },
+  {
+    title: 'PayPal invested $1700 in MercuryCorp and $1100 in JupiterCorp',
+    content: 'Ut enim ad minim veniam, quis nostrud exercitation ullamco JupiterCorp laboris nisi ut aliquip ex ea commodo bitcoin consequat. Duis aute irure MercuryCorp dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.',
+    platform: "facebook",
+    image: 'https://upload.wikimedia.org/wikipedia/fr/f/fc/Reddit-alien.png',
+    date: new Date('November 26, 2020 19:29:24'),
+    popularity: 999,
+  },
+  {
+    title: 'PayPal invested $2700 in MercuryCorp and $1100 in JupiterCorp',
+    content: 'Ut enim ad minim veniam, quis nostrud exercitation ullamco JupiterCorp laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure MercuryCorp dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.',
+    platform: "facebook",
+    image: 'https://upload.wikimedia.org/wikipedia/fr/f/fc/Reddit-alien.png',
+    date: new Date('November 14, 2020 14:22:27'),
+    popularity: 540,
+  },
+  {
+    title: 'PayPal invested M$3900 in MercuryCorp and $2100 in JupiterCorp',
+    content: 'Ut enim ad minim bitcoin veniam, quis nostrud exercitation ullamco JupiterCorp laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure MercuryCorp dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.',
+    platform: "twitter",
+    image: 'https://upload.wikimedia.org/wikipedia/fr/f/fc/Reddit-alien.png',
+    date: new Date('May 15, 2020 16:27:27'),
+    popularity: 902,
+  },
 
 ]
 
-module.exports = mentions
```

Comment:
Why did you remove the export for this file?

Reply:
Actually it's showing on line 87. I did add some new Mentions in the file.